airflow-core/src/airflow/migrations/versions/0041_3_0_0_rename_dataset_as_asset.py

```diff
@@ -301,9 +348,12 @@ def upgrade():
         if op.get_bind().dialect.name in ("postgresql", "mysql"):
             batch_op.drop_constraint("ddrq_dataset_fkey", type_="foreignkey")
 
-        _rename_pk_constraint(
+    with op.batch_alter_table("asset_dag_run_queue", schema=None) as batch_op:
+        _rename_pk_constraint_unkown(
             batch_op=batch_op,
-            original_name="datasetdagrunqueue_pkey",
+            table_name="asset_dag_run_queue",
+            original_name="dataset_dag_run_queue_pkey",
+            alternative_name="datasetdagrunqueue_pkey",
```

Comment:
why is this better?

Reply:
Slightly shorter I guess?

Reply:
In Airflow 2, these were different names for ORM and migration files. If users upgraded through migration file, they would have one of the names. If they created the db from ORM, they would also have one of the names. So I did it this way so that no matter the name that came up, we could drop it and rename it to the new name